website/source/docs/guides/minikube.md

```diff
@@ -0,0 +1,198 @@
+
+First, you'll need to follow the directions for [installing minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/), including VirtualBox or similar, kubectl, and minikube. You'll also need to install `helm` which is available at [helm.sh](https://helm.sh/).
+
+Mac users can install `helm` and `kubectl` with Homebrew.
+
+```sh
+$ brew install kubernetes-cli
+$ brew install kubernetes-helm
+```
+
+## Task 1: Start Minikube and Install Consul with Helm
+
+Next, start minikube. I like to use the `--memory` option with the equivalent of 4GB to 8GB so there is plenty of memory for all the pods we will run. This may take a while...on my machine it had to download a few hundred MB of dependencies and container images.
+
+```
+minikube start --memory 4096
+```
+
+Next, let's start the local Kubernetes dashboard with `minikube dashboard`. Even if the previous step completed successfully, you may have to wait a minute or two for minikube to be available. If you see this message, try again.
+
+Once it spins up, you'll see the dashboard in your web browser. You can view pods, nodes, and other resources.
+
+```
+minikube dashboard
+```
+
+To perform the steps in this lab exercise, clone the `hashicorp/demo-consul-101` repository from GitHub. Change into the repo, and go to the `k8s` directory inside.
+
+
+```
+git clone https://github.com/hashicorp/demo-consul-101.git
+
+cd demo-consul-101
+```
+
+Now we're ready to install Consul to the cluster, using the `helm` tool. Initialize Helm with `helm init`. You'll see a note that Tiller (the server-side component) has been installed. You can ignore the policy warning.
+
+```
+helm init
+```
+
+Now we need to install Consul with Helm. To get the freshest copy of the Helm chart, clone the `hashicorp/consul-helm` repository.
+
+```
+git clone https://github.com/hashicorp/consul-helm.git
+```
+
+The chart works on its own, but we'll override a few values to help things go more smoothly with minikube and to enable useful features.
+
+We've created `helm-consul-values.yaml` for you with overrides. See `values.yaml` in the Helm chart repository for other possible values.
+
+We've given a name to the datacenter running this Consul cluster. We've enabled the Consul web UI. In this case, it specifies that a load balancer is needed. This will only work when deployed to a hosted cloud that implements load balancers, which minikube does not. You could use a NodePort here or leave it as-is and we'll access the Consul UI in another way. We'll enable secure communication between pods with Connect. We also need to enable grpc on the client for this to work properly. Finally, specify that this Consul cluster should only have one server (suitable for local development).
+
+```yaml
+global:
+  datacenter: minidc
+
+ui:
+  service:
+    type: "LoadBalancer"
```

Comment:
Since we aren't running on a cloud platform, we can just use `NodePort`. 
```suggestion
    type: "NodePort"
```

Reply:
Fixed in 607fb9e78